aeon/dj_pipeline/report.py

```diff
@@ -45,14 +45,13 @@ class InArenaSummaryPlot(dj.Computed):
     }
 
     def make(self, key):
-        raw_data_dir = acquisition.Experiment.get_data_directory(key)
-
         in_arena_start, in_arena_end = (
             analysis.InArena * analysis.InArenaEnd & key
         ).fetch1("in_arena_start", "in_arena_end")
 
         # subject's position data in the time_slices
         position = analysis.InArenaSubjectPosition.get_position(key)
+        position.rename({"position_x": "x", "position_y": "y"}, inplace=True)
```

Comment:
should be changed to:

`position.rename(columns={"position_x": "x", "position_y": "y"}, inplace=True)`

Reply:
yes, throws this error without specifying it.

> Exception has occurred: IndexError
> too many indices for array
>   File "/home/jaerong/ProjectAeon/aeon_mecha/aeon/dj_pipeline/report.py", line 54, in make
>     position.rename({"position_x": "x", "position_y": "y"}, inplace=True)
>   File "/home/jaerong/ProjectAeon/code/analysis.py", line 13, in <module>
>     InArenaSummaryPlot.populate(limit=1)